src/js/containers/agency/AgencyContainerV2.jsx

```diff
@@ -65,72 +68,41 @@ const componentByAgencySection = {
     top_5_award_dimensions: <ComingSoonSection section="top_5_award_dimensions" />
 };
 
-const sankeyRef = createRef();
-
-const verticalOffsetExpanded = 266;
-const verticalOffsetCollapsed = 121;
+let timeout;
 
 export const AgencyProfileV2 = ({
     agencyOverview,
     agencyId,
+    params,
     clearAgency,
     setOverview
 }) => {
     const [activeSection, setActiveSection] = useState('overview');
-    const [isSankeyExpanded, setSankeyExpanded] = useState(true);
-    // height in px of element's w/ a fixed position (sankey + header)
-    const [verticalOffset, setVerticalOffset] = useState(verticalOffsetExpanded);
-    const [selectedFy, setSelectedFy] = useState(`${FiscalYearHelper.defaultFiscalYear()}`);
-
-    const getSectionsWithVerticalOffset = (offset) => Object.keys(componentByAgencySection)
-        .map((section, i) => ({
-            section,
-            label: startCase(section),
-            stickyVerticalOffset: offset
-        }));
-
-    useEffect(() => {
-        if (isSankeyExpanded && verticalOffset !== verticalOffsetExpanded) {
-            setVerticalOffset(verticalOffsetExpanded);
-        }
-        else if (!isSankeyExpanded && verticalOffset !== verticalOffsetCollapsed) {
-            setVerticalOffset(verticalOffsetCollapsed);
+    const [showConfirmationText, setConfirmationText] = useState(false);
+    const [selectedFy, setSelectedFy] = useState(
+        `${FiscalYearHelper.defaultFiscalYear()}`
+    );
+
+    useEffect(() => () => {
+        if (timeout && showConfirmationText) {
+            window.clearTimeout(timeout);
         }
-    }, [verticalOffset, setVerticalOffset, isSankeyExpanded]);
-
-    const sortFy = (a, b) => {
-        if (a === selectedFy) return -1;
-        if (b === selectedFy) return 1;
-        if (b > a) return 1;
-        if (a > b) return -1;
-        return 0;
+    });
+
+    const getCopyFn = () => {
+        document.getElementById('slug').select();
+        document.execCommand("copy");
+        setConfirmationText(true);
+        timeout = window.setTimeout(() => {
```

Comment:
interesting, I am wondering if we could implement something other than the timeout. Maybe the message be a modal so they can x out if it themselves or if the click anywhere it goes away.